bucket/ultravnc.json

```
@@ -0,0 +1,45 @@
+{
+    "version": "1.3.6.0",
+    "description": "UltraVNC Server and Viewer can display the screen of one computer (Server) on the screen of another (Viewer).",
+    "homepage": "https://uvnc.com",
+    "license": "GPL-3.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://uvnc.com/component/jdownloads/send/0-/420-ultravnc-1-3-60-bin-zip.html?Itemid=0#/dl.zip",
+            "hash": "c2c36c1b088ebe06d7bc6f6b4bef603f051a33350a628ff46e20e4beb2f906f2",
+            "extract_dir": "x64"
+        },
+        "32bit": {
+            "url": "https://uvnc.com/component/jdownloads/send/0-/420-ultravnc-1-3-60-bin-zip.html?Itemid=0#/dl.zip",
+            "hash": "c2c36c1b088ebe06d7bc6f6b4bef603f051a33350a628ff46e20e4beb2f906f2",
+            "extract_dir": "x86"
+        }
+    },
+    "shortcuts": [
+        [
+            "vncviewer.exe",
+            "VNC Viewer"
+        ],
+        [
+            "winvnc.exe",
+            "WinVNC"
+        ]
+    ],
+    "checkver": {
+        "url": "https://uvnc.com/downloads/ultravnc/",
+        "re": "/downloads/ultravnc/([\\d])-ultravnc-([\\d])-([\\d])-([\\d])-([\\d])\\.html",
+        "replace": "${2}.${3}.${4}.${5}"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://uvnc.com/component/jdownloads/send/0-/${1}-ultravnc-${2}-${3}-${4}${5}-bin-zip.html?Itemid=0#/dl.zip",
+                "extract_dir": "x64"
```

```suggestion
                "url": "https://uvnc.com/component/jdownloads/send/0-/${1}-ultravnc-${2}-${3}-${4}${5}-bin-zip.html?Itemid=0#/dl.zip"
```

Updating extract_dir is not required